config/environments/staging.rb

```diff
@@ -3,7 +3,20 @@
 CodeMontage::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb
 
-  config.action_mailer.default_url_options = { host: 'codemontage.com' }
+  config.action_mailer.default_url_options = { host: 'codemontage-staging.herokuapp.com' }
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.